Anytype/Sources/PresentationLayer/Flows/EditorFlow/EditorCoordinatorView.swift

```diff
@@ -7,9 +7,31 @@ struct EditorCoordinatorView: View {
     @Environment(\.pageNavigation) private var pageNavigation
     
     var body: some View {
-        model.makeView()
+        mainView
             .onAppear {
                 model.pageNavigation = pageNavigation
             }
     }
+    
+    @ViewBuilder
```

Comment:
redundant view builder?

Reply:
Yes. Method return view with different types. Compiler make a error without view builder, because it can't resolve `any View` to one specific type. ViewBuilder make a container and fix error